src/components/pages/Footer.astro

```diff
@@ -254,6 +270,21 @@ import WmNamedLogo from '@assets/wm-named-logo.svg'
     border: none;
   }
 
+  .copyright {
+    justify-self: center;
```

Comment:
This isn't centered on larger screens.

<img width="2951" height="651" alt="image" src="https://github.com/user-attachments/assets/f4e79728-08dc-495f-ba81-02912db6756e" />

<img width="1093" height="400" alt="image" src="https://github.com/user-attachments/assets/412cd0b4-b20c-4c01-a31b-be0219998006" />